app.py

```diff
@@ -0,0 +1,75 @@
+from bson.json_util import dumps
+from bson.objectid import ObjectId
+from flask import Flask, render_template, request, redirect, url_for
+from pymongo import MongoClient
+
+import datetime
+import json
+import os
+import sys
+
+client = MongoClient(os.environ['MONGODB_URI'])
+db = os.environ['MONGODB_URI'].split('/')[-1]
```

Comment:
Believe there's no need to manually split out the DB here - can use `client.get_database()` w/o args to get the db named in the URI.

Reply:
Very nice